Sources/TuistAsyncQueue/AsyncQueuePersistor.swift

```diff
@@ -70,12 +76,21 @@ final class AsyncQueuePersistor: AsyncQueuePersisting {
                 try? await fileSystem.remove(eventPath)
                 continue
             }
+
```

Comment:
I noticed we do `for eventPath in paths` processing events sequentially. What do you think about processing them in parallel to get a bit of performance improvement?